sdk/include/wdf/kmdf/1.9/wdfpool.h

```diff
@@ -0,0 +1,31 @@
+/*
+ * wdfpool.h
+ *
+ * Windows Driver Framework - Pool API's
+ *
+ * This file is part of the ReactOS WDF package.
+ *
+ * Contributors:
+ *   Created by Benjamin Aerni <admin@bennottelling.com>
+ *
+ * Intended Usecase:
+ *   Kernel mode drivers
+ *
+ * THIS SOFTWARE IS NOT COPYRIGHTED
+ *
+ * This source code is offered for use in the public domain. You may
+ * use, modify or distribute it freely.
+ *
+ * This code is distributed in the hope that it will be useful but
+ * WITHOUT ANY WARRANTY. ALL WARRANTIES, EXPRESS OR IMPLIED ARE HEREBY
+ * DISCLAIMED. This includes but is not limited to warranties of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.
+ *
+ */
+#ifndef _WDFPOOL_H_
+#define _WDFPOOL_H_
+
+
+
```

Comment:
Empty ??

Reply:
MS also has this header empty, but has kept it all these years.